packages/components/pie-icon-button/README.md

```diff
@@ -8,37 +8,137 @@
   </a>
 </p>
 
+# Table of Contents
+
+1. [Introduction](#pie-icon-button)
+2. [Local Development](#local-development)
+3. [Props](#props)
+4. [Events](#events)
+   - [HTML example](#html)
+   - [Vue example (using Nuxt 3)](#vue-templates-using-nuxt-3)
+   - [React example (using Next 13)](#react-templates-using-next-13)
+5. [Testing](#testing)
+   - [Browser Tests](#browser-tests)
+   - [Visual Tests](#visual-tests)
+
+
 ## `pie-icon-button`
 
 `pie-icon-button` is a Web Component built using the Lit library. It offers a simple and accessible icon button component for web applications. This component can be easily integrated into various frontend frameworks and customized through a set of properties.
 
+## Local development
+
+Install the dependencies from the root of the monorepo:
+
+```bash
+yarn
+```
+
+To build the `pie-icon-button` package, run the following command:
+
+```bash
+yarn build --filter=pie-icon-button
+```
+
+If you'd like to develop using the component storybook, then you should build the component in `watch` mode, and run storybook in a separate terminal tab:
+
+
+```bash
+yarn watch --filter=pie-icon-button
+
+# in a separate terminal tab, run
+yarn dev --filter=pie-storybook
+```
+
+
+### Importing the component
+
+```js
+// Default – for Native JS Applications, Vue, Angular, Svelte etc.
+import { PieIconButton } from '@justeattakeaway/pie-icon-button';
+
+// React
+// For React, you will need to import our React specific component build
+// Which wraps the web component using the @lit-labs/react package
+import { PieIconButton } from '@justeattakeaway/pie-icon-button/dist/react';
+```
+
+
+## Props
+
+| Property    | Type      | Default         | Description                                                          |
+|-------------|-----------|-----------------|----------------------------------------------------------------------|
+| size        | `String`  | `medium`        | Size of the Icon Button, one of `iconButtonSizes` – `xsmall | small | medium | large` |
+| variant     | `String`  | `primary`       | Variant of the button, one of `iconButtonVariants` – `primary | secondary | outline | ghost | ghost-tertiary` |
```

Comment:
Using pipes messes up the table formatting
```suggestion
| size        | `String`  | `medium`        | Size of the Icon Button, one of `iconButtonSizes` – `'xsmall', 'small', 'medium', 'large'` |
| variant     | `String`  | `primary`       | Variant of the button, one of `iconButtonVariants` – `'primary', 'secondary', 'outline', 'ghost', 'ghost-tertiary'` |
```

Reply:
Done